composer.json

```diff
@@ -17,9 +17,8 @@
     }
   },
   "require": {
-    "php": ">=5.5.9",
```

Comment:
What's the reason we are removing instead of replacing the version?

Reply:
Good point, I got confused what I was removing there